javascript/modules/client/src/client.ts

```diff
@@ -546,27 +549,48 @@ export class Client extends ClientCore implements IClient {
       override.value = amount;
     }
 
-    // TODO: If the approved ERC20 amount is not enough,
-    // estimate the cose of increasing the allowance
-
     return daoInstance.estimateGas
```

Comment:
Why are we removing the TODO without implementing it?

Reply:
I believe this comment was before we knew the UI will be dividing the deposit in two steps, so since we now have the estimate of increasing allowance i believe it does not make sense but let me know what you think

Reply:
The point is that the SDK should make sense to anyone, not just to what the UI is deciding to do in UX terms. If we only leave this functionality, we should have two clearly distinct functions where names make it obvious that only one part is estimated. But personally, I prefer an SDK where the exact same thing that will be executed in real life is the one that we will estimate

Reply:
I mean, for me the way it is right now is pretty clear and does not need changes:
- `estimation.deposit` -> estimates the gas fees for making a deposit
- `estimation.updateAllowance` -> estimates the gas fees for updating the allowance